src/DiagramGenerator/Diagram/Board.php

```diff
@@ -380,11 +380,12 @@ protected function getMaxPieceHeight()
      */
     protected function cacheImage($remoteImageUrl, $cachedFilePath)
     {
+        $cachedFilePathTmp = $cachedFilePath . rand(1000000,9999999);
```

Comment:
Even in the worst case, this will be a thousand times faster than the HTTP call coming up, so I'm not worried about performance here. But you are right that I don't actually want a random file name, but instead I want a unique one. It's better if the code reflected that. :+1: